src/main/java/org/opentripplanner/routing/graph_finder/DirectGraphFinder.java

```diff
@@ -0,0 +1,65 @@
+package org.opentripplanner.routing.graph_finder;
```

Comment:
FYI: We have a few of these package names with `_` in them, but strictly they are not recommended by the original Java Style guide. I might refactor it at some point to be compliant. The standard recommend using just `graphfinder`. I guess the reason is quite old; some filesystems having problems with special characters.

Reply:
Renamed, easier to do now.